addons/launcher-addon/src/main/java/io/fabric8/launcher/addon/ui/booster/ProjectInfoStep.java

```diff
@@ -214,9 +219,7 @@ public Result execute(UIExecutionContext context) throws Exception {
             if (runtimeVersion.getValue() != null) {
                 profileId = runtimeVersion.getValue().getId();
             }
-            if (booster.getBuildProfile() != null) {
-                profileId = booster.getBuildProfile();
-            }
+            booster.getMetadata("buildProfile", profileId);
```

Comment:
Shouldn't the return of this method be assigned to `profileId` ?

Reply:
Good catch, thx!